identity/identity.py

```diff
@@ -0,0 +1,138 @@
+import json
+import re
+import time
+
+import requests
+from django.core.exceptions import ImproperlyConfigured
+from jwcrypto import jwk, jwt
+
+from accounts.settings import accounts_settings
+
+
+JWKS_URL = f"{accounts_settings.PLATFORM_URL}/identity/jwks/"
+ATTEST_URL = f"{accounts_settings.PLATFORM_URL}/identity/attest/"
+REFRESH_URL = f"{accounts_settings.PLATFORM_URL}/identity/refresh/"
+
+
+class IdentityContainer:
+    refresh_jwt: jwt.JWT = None
+    access_jwt: jwt.JWT = None
+    platform_jwks: jwk.JWKSet = None
+
+
+container = IdentityContainer()
+
+
+def get_platform_jwks():
+    """
+    Download the JWKS from Platform to verify JWTs
+    """
+
+    try:
+        response = requests.get(JWKS_URL)
+        # For some reason this method wants a raw string instead of a python dictionary
+        container.platform_jwks = jwk.JWKSet.from_json(response.text)
+    except Exception:
+        container.platform_jwks = None
+
+
+def attest():
+    """
+    Perform the initial authentication (attest) with Platform using the Client ID
+    and Secret from DOT
+    """
+
+    response = requests.post(
+        ATTEST_URL, auth=(accounts_settings.CLIENT_ID, accounts_settings.CLIENT_SECRET)
+    )
+    if response.status_code == 200:
+        content = response.json()
+        container.access_jwt = jwt.JWT(key=container.platform_jwks, jwt=content["access"])
+        container.refresh_jwt = jwt.JWT(key=container.platform_jwks, jwt=content["refresh"])
+        return True
+    return False
```

Comment:
`attest()` is also run on startup (from `identity/apps.py`). If we raise an exception immediately it would be more of a "compile-time" check, but by only throwing it when making a request it becomes a runtime check. Since not every product is going to be making b2b requests, I felt this made more sense, but happy to discuss more